features/step_definitions/user_steps.rb

```diff
@@ -29,7 +29,7 @@
 end
 
 Given "an active user {string} of {account} with email {string}" do |username, account, email|
-  FactoryBot.create(:active_user, :account => account, :username => username, :email => email)
+  @user = FactoryBot.create(:active_user, :account => account, :username => username, :email => email)
```

Comment:
```suggestion
  @user = FactoryBot.create(:active_user, account: account, username: username, email: email)
```

As we're touching this line anyway...